ci/test/00_setup_env_i686_centos.sh

```diff
@@ -9,7 +9,8 @@ export LC_ALL=C.UTF-8
 export HOST=i686-pc-linux-gnu
 export CONTAINER_NAME=ci_i686_centos
 export DOCKER_NAME_TAG=quay.io/centos/centos:stream8
-export DOCKER_PACKAGES="gcc-c++ glibc-devel.x86_64 libstdc++-devel.x86_64 glibc-devel.i686 libstdc++-devel.i686 ccache libtool make git python3 python3-zmq which patch lbzip2 xz procps-ng dash rsync coreutils bison"
+export DOCKER_PACKAGES="gcc-c++ glibc-devel.x86_64 libstdc++-devel.x86_64 glibc-devel.i686 libstdc++-devel.i686 ccache libtool make git python3 python3-pip zeromq which patch lbzip2 xz procps-ng dash rsync coreutils bison"
+export PIP_PACKAGES="pyzmq"
```

Comment:
How does this PR change the "32-bit + dash [gui] [CentOS 8]" CI task?

Asking because the `interface_zmq.py` functional test [passes](https://cirrus-ci.com/task/5464815220031488) on the current master branch.

Reply:
It shouldn't. All it does is replace the `python3-zmq` system package, which has wrapped the `pyzmq` from pip and `zeromq`. It is replaced by an explicit installation of both "sub"-packages.

Reply:
Then, probably, the relevant docs should be adjusted as well?

Reply:
> All it does is replace the `python3-zmq` system package...

Other tasks still use `python3-zmq`.

Reply:
> apt-get install python3-zmq

centos is not using `apt-get`, so any doc update seems unrelated. If your system provides `python3-zmq` you can use it, however, you can also use `pip3 install pyzmq`, which should work everywhere.

Reply:
> centos is not using `apt-get`, so any doc update seems unrelated

That is why docs need to be updated like that o similar:
```
The ZMQ functional test requires a python ZMQ library. To install it:
- on Debian/Ubuntu, run `sudo apt-get install python3-zmq`
- on other systems, run `pip3 install pyzmq`
```

But it is unrelated to this PR CI changes themself.

---

> All it does is replace the `python3-zmq` system package, which has wrapped the `pyzmq` from pip and `zeromq`. It is replaced by an explicit installation of both "sub"-packages.

And what are benefits of that?

Reply:
> Also, it is needed for other changes (bumping the minimum python version).

Perhaps, mention this in the PR description?

Reply:
> Perhaps, mention this in the PR description?

Thx, done